src/evict/evict_lru.c

```diff
@@ -907,24 +907,7 @@ __evict_lru_walk(WT_SESSION_IMPL *session)
 	/* Get some more pages to consider for eviction. */
 	if ((ret = __evict_walk(cache->walk_session, queue)) == EBUSY)
 		return (0);	/* An interrupt was requested, give up. */
-	WT_RET_NOTFOUND_OK(ret);
-
-	/*
-	 * If we found no pages at all during the walk, something is wrong.
-	 * Be more aggressive next time.
-	 *
-	 * Continue on to sort the queue, in case there are pages left from a
-	 * previous walk.
-	 */
-	if (ret == WT_NOTFOUND) {
-		if (F_ISSET(cache,
-		    WT_CACHE_EVICT_CLEAN_HARD | WT_CACHE_EVICT_DIRTY_HARD))
-			cache->evict_aggressive_score = WT_MIN(
-			    cache->evict_aggressive_score + WT_EVICT_SCORE_BUMP,
-			    WT_EVICT_SCORE_MAX);
-		WT_STAT_CONN_SET(session, cache_eviction_aggressive_set,
-		    cache->evict_aggressive_score);
-	}
```

Comment:
I'm happy for that part of the change to be reverted.  Early on, some of the wtperf workloads were much slower when we got aggressive, which motivated this change.  It may not be necessary at all given the other changes in this branch, and I'd prefer to get the fix in asap.

I can't do the revert now: Monday is the earliest I'll get to it.